integrations/quick-start-guide.md

```diff
@@ -0,0 +1,47 @@
+
+## Add an Integration
+To add an integration, simply click the icon for the integration you would like to connect. You would need to provide your service details together with your credentials. You could find more details [here](./adding-an-integration/) on connecting to s3, bigquery, and Google Cloud Storage (GCS).
+
+Once you added an integration, you can refer to [using integrations guide](./using-integrations/) to use the integration in Aqueduct SDK.
+
+## Connected Integrations
+You can view a list of all connected integrations. You can click into each item to view more details or manage that integration. Every Aqueduct server comes with a [demo integration](./aqueduct-demo-integration.md) that you can play around with.
+
+# Integration Detail Page
+In this page, you can view details about an integration and make changes to it:
+* [Integration Information](#integration-information "mention")
+* [Test-Connecting an Integration](#test-connecting-an-integration "mention")
+* [Editing an Integration](#editing-an-integration "mention")
+* [Deleting an Integration](#deleting-an-integration "mention")
+* [Viewing Data in an Integration](#viewing-data-in-an-integration "mention")
+* [Viewing Workflows Using an Integration](#viewing-workflows-using-an-integration "mention")
+
+## Integration Information
+At top of the integration detail page. You can view basic information about this integration. There are also a number of options to manage it, including [test-connection](#test-connecting-an-integration "mention"), [edit the integration](#editing-an-integration "mention"), and [delete the integration](#deleting-an-integration "mention").
+![](<../.gitbook/assets/integration_information.png>)
+
+## Test-Connecting an Integration
+In `Options` menu, you can test-connect an integration to verify if it's currently available. This is especially helpful to confirm whether your credentials are expired.
+
+## Editing an Integration
+In `Options` menu, you can edit an integration to update its name and credentials. The changes will also update all existing workflows using this integration.
+
+For now, we do not support updating fields that requires potential data migration. For example, `host` and `database` for relational databases. To make changes on these fields, you would need to follow a migration process:
+1. Create a new integration.
+2. Perform any necessary data migration in your integration.
+3. [Update your workflows](../workflows/editing-a-workflow.md) to use the new integration.
+
+## Deleting an Integration
+If you don't have any workflow using an integration, you can delete that integration from `Options` menu. Otherwise, you would need to [delete all workflows using this integration](../workflows/deleting-a-workflow.md) first.
```

Comment:
Actually, as a nice-to-have task, maybe we can include the workflow DAG version # or something in the workflow dropdown since we don't allow an integration to be deleted if it is used in any workflow DAG, active or not and I think the workflow is/should be showing operators using the integration even if it is not the currently active DAG. It might be confusing for the user since they'll see the integration is currently used by a workflow then see the latest workflow DAG and not see the integration.

Reply:
Actually the current behavior is we show active version at glance and there's a drop-down button to show older versions, so both current / old versions are reflected in the UI. But I agree ideally we could provide more version information here (like link to actual DAG version rather than just the wf) and the UX could be more polished.